src/wallet/wallet_rpc_server_commands_defs.h

```diff
@@ -867,7 +867,7 @@ namespace wallet_rpc
       std::string address;    // Destination public address.
       uint32_t priority;      // Set a priority for the transaction. Accepted values are: or 0-5 for: default, unimportant, normal, elevated, priority, blink.
       bool blink;             // (Deprecated) Set priority to 5 for blink, field is deprecated: specifies that the tx should be blinked (`priority` will be ignored).
-      uint64_t ring_size;     // (Deprecated) Set to 10. Sets ringsize to n (mixin + 1). Loki ring_size is statically set to 10.
+      uint64_t ring_size;     // (Deprecated) Ignored; Loki ring_size is statically set to 10.
```

Comment:
Now would be a good time to delete these from the request, as well as `blink`

Reply:
Don't we still have wallets out there sending `blink` to blink?

Reply:
Don't we still have early 6.x wallets using `blink=1`?  Or all they *also* setting the priority?

Reply:
Yeah. Guess I wasn't thinking of that case, Android and IOS are ok with auto updating but old gui/cli wallets might still be out there.

Could you also go and update src/wallet/api/wallet.cpp mixin and priority code?

Reply:
Updated.